hello-oboe/app/src/main/cpp/CMakeLists.txt

```diff
@@ -13,46 +13,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 cmake_minimum_required(VERSION 3.4.1)
+project(hello-oboe LANGUAGES CXX)
 
-# Pick up a stable Oboe lib version to use:
-# in the future, will point to a release branch rather than a commit hash
-set(OBOE_VERSION bb066c330a201913bd18686901381ad820c41296)
-
-### INCLUDE OBOE LIBRARY ###
-get_filename_component(OBOE_SAMPLE_PROJ_HOME
-        ${CMAKE_CURRENT_SOURCE_DIR}/.. ABSOLUTE)
-set(OBOE_SRC_DIR ${OBOE_SAMPLE_PROJ_HOME}/oboe)
-
-# clone the dependency repo.
-# git submodule could also be used if this sample does not need
-#     Android Studio's "Import Android code sample" option
-if ((NOT EXISTS ${OBOE_SRC_DIR}) OR
-(NOT EXISTS ${OBOE_SRC_DIR}/CMakeLists.txt))
-    execute_process(COMMAND git clone https://github.com/google/oboe.git oboe
-            WORKING_DIRECTORY ${OBOE_SAMPLE_PROJ_HOME}/)
-    execute_process(COMMAND git checkout ${OBOE_VERSION}
-            WORKING_DIRECTORY ${OBOE_SAMPLE_PROJ_HOME}/oboe)
-endif()
-
-# Add the Oboe library as a subproject. Since Oboe is an out-of-tree source library we must also
-# specify a binary directory
-add_subdirectory(${OBOE_SRC_DIR} ./oboe-bin)
-
-# Include the Oboe headers
-include_directories(${OBOE_SRC_DIR}/include)
-
-### END OBOE INCLUDE SECTION ###
-
-# Build the libhello-oboe library
-add_library(hello-oboe SHARED
-        hello-oboe.cpp
-        )
+# add oboe pre-release lib hosted at https://maven.google.com/web/index.html
+# under com.google.oboe:oboe. For documentation about oboe pre-built lib, refer to
+# https://github.com/google/oboe/blob/master/docs/GettingStarted.md#option-1-using-pre-built-binaries-and-headers
+find_package(oboe REQUIRED CONFIG)
```

Comment:
This will prevents us from using new APIs in the sample code before the API changes have been released. I think maybe the samples should continue to link directly with the source code in this repo.  Maybe we should have a sample not in this repo that uses the pre-built, like the NDK samples.